sympy/printing/mathml.py

```diff
@@ -554,6 +554,8 @@ def multiply(expr, mrow):
             numer, denom = fraction(expr)
             if denom is not S.One:
                 frac = self.dom.createElement('mfrac')
+                if self._settings["fold_short_frac"]:
```

Comment:
Should one make some sort of length test here? Or does one already know that the length is short?

Reply:
Oh right. I forgot it there.